### ISSUE TITLE
[Accessibility] [Advisory] [Navigation] nav element has unnecessary navigation role

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1669]
**Steps to Reproduce**
1. Validate markup using the W3C Nu HTML Checker; note warnings.

**Current Behavior**
The `<nav>` element wrapping the navigation component has a redundant `role="navigation"` applied.

**Expected behavior/code**
The redundant role attribute should be removed from the  `<nav>` element, which has a role of navigation by default.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Assistive technology users

**WCAG SC**
4.1.2

**Possible Solution**
Remove role attribute from `<nav>` element.

**Code snippet**
```
<nav id="navigation-0e123bccd2" class="cmp-navigation" role="navigation" itemscope="" itemtype="http://schema.org/SiteNavigationElement" data-cmp-data-layer="{&quot;navigation-0e123bccd2&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/navigation&quot;,&quot;repo:modifyDate&quot;:&quot;2019-05-21T15:59:35Z&quot;}}">
```

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by bpauli** [comment]
https://github.com/adobe/aem-core-wcm-components/pull/1696
